adcm_aio_client/core/client.py

```diff
@@ -24,15 +24,15 @@ def __init__(self: Self, requester: Requester) -> None:
 
     @cached_property
     def clusters(self: Self) -> ClustersNode:
-        return ClustersNode(path=(), requester=self._requester)
+        return ClustersNode(path=("clusters",), requester=self._requester)
```

Comment:
why passing this path is required now?

Reply:
it wasnt? without path this nodes do not work